demisto_sdk/commands/unify/tests/unifier_test.py

```diff
@@ -81,8 +81,10 @@ def __init__(self, tenant_id: str = '', auth_id: str = '', enc_key: str = '',
 TESTS_DIR = f'{git_path()}/demisto_sdk/tests'
 
 
-def test_clean_python_code():
-    unifier = Unifier("test_files/VulnDB")
+def test_clean_python_code(repo):
+    pack = repo.create_pack('PackName')
+    integration = pack.create_integration('integration', 'bla', INTEGRATION_YAML)
+    unifier = Unifier(str(integration.path))
```

Comment:
why did you change this?

Reply:
This path did not really exists, thats why os.path.isdir failed. so I created a mock integration just for the path to indeed exist.